tests/test_patron.py

```diff
@@ -0,0 +1,308 @@
+import datetime
+
+from bs4 import BeautifulSoup
+from freezegun import freeze_time
+
+from patronload.patron import (
+    format_phone_number,
+    patron_xml_from_records,
+    populate_patron_common_fields,
+)
+
+SIX_MONTHS = "2023-09-01Z"
+TWO_YEARS = "2025-09-01Z"
+
+
+def test_format_phone_number_valid_value_succeeds():
+    assert format_phone_number("1111111111") == "111-111-1111"
+
+
+def test_format_phone_number_invalid_value_is_returned():
+    assert format_phone_number("abcd") == "abcd"
+
+
+@freeze_time("2023-03-01")
+def test_patron_xml_from_records_staff_success():
```

Comment:
A couple things for this test:

1. I would include a record with null values as one of these, and I would probably have it processed first to confirm the issue from my previous comment is resolved.
2. These variable names are pretty confusing because they're extremely similar and don't convey much about what they actually are. I would suggest using variables like: `expected_staff_xml_output_from_null_values`, `staff_database_record_with_null_values`, and `result` for the final result that is being tested.

Reply:
I'm actually adding in the reject functionality, it doesn't make sense to test this function without it and can be expanded upon later